src/composables/multisigTransactions.ts

```diff
@@ -88,7 +88,7 @@ export function useMultisigTransactions({ store }: IDefaultComposableOptions) {
         hash: txHash,
         tx: rawTx ? unpackTx(rawTx.tx) as any : undefined,
         isMultisigTransaction: true,
-        microTime: rawTx ? new Date(rawTx.createdAt).getTime() : undefined,
```

Comment:
The `createdAt` property actually contains the time when the vault was created, not the proposal.

Reply:
So we are setting the `microTime` to be `Date.now` in order to have the pending multisig transactions on top?

Reply:
Yes, this is what we exactly doing on adding a regular pending transaction.